docs/development_environment.mdx

```diff
@@ -59,6 +59,24 @@ If the Dev Container was set up correctly - it supports debugging by default, ou
 
 It is also possible to debug a remote Home Assistance instance (e.g., production instance) by following the procedure described [here](https://www.home-assistant.io/integrations/debugpy/).
 
+### Use devcontainer from WSL
+
+With WSL, you won't run into as many path issues because of the '/' and '\' differences between Linux and Windows. 
+This is handy when working on frontend code and using `script/develop`, since it's using stuff that works best on Linux.
```

Comment:
we do not support native Windows at all, so this sentence feels unnecessary :thinking:

Reply:
I've reworded it to focus more on the benefits on this approach.